wagtail_localize/models.py

```diff
@@ -461,3 +459,11 @@ def set_locale_on_new_instance(sender, instance, **kwargs):
         return
 
     instance.locale_id = instance.get_default_locale_id()
+
+
+@receiver(post_save, sender=Locale)
```

Comment:
This would fire the signals whenever the `Locale` is saved. I guess we probably won't be saving `Locale` for any other reason than to activate/deactivate them, but is there a way for us to detect if `is_active` has changed?

Reply:
Maybe we could do a check in the `pre_save` signal, but we would still need to fire in `post_save`

Alternatively, we could probably add a separate field that stores the last signalled state? If the field ever defers with `is_active` we know to fire the signal